### PR TITLE
add healthchecks pings to pg backup cronjobs

### PR DESCRIPTION
**Pull request by atvaccaro**
# Description
_Describe your changes and why you're making them. Please include the context, motivation, and relevant dependencies._

Resolves https://github.com/cal-itp/data-infra/issues/2671

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

## How has this been tested?
_Include commands/logs/screenshots as relevant._

See https://healthchecks.jarv.us! Lmk if you need an invite.

## Post-merge follow-ups
_Document any actions that must be taken post-merge to deploy or otherwise implement the changes in this PR (for example, running a full refresh of some incremental model in dbt). If these actions will take more than a few hours after the merge or if they will be completed by someone other than the PR author, please create a dedicated follow-up issue and link it here to track resolution._

- [x] No action required
- [ ] Actions required (specified below)

### CODE REVIEW

File: kubernetes/apps/charts/postgresql-backup/values.yaml [hunk 1]
```diff
@@ -7,3 +7,6 @@ secretMounts: []
 config:
   envFrom: []
   env: []
+
+healthchecks:
+  host: https://healthchecks.ops.k8s.jarv.us
```
Comment (SorenSpicknall) [inline kubernetes/apps/charts/postgresql-backup/values.yaml:12]: Should there be a `calitp.org` host configured for this?

**Review by SorenSpicknall — APPROVED** [review]
Just one question RE: host

File: kubernetes/apps/charts/postgresql-backup/templates/cronjob.yaml [hunk 2]
```diff
@@ -53,10 +57,29 @@ spec:
...
+                  prune_status=$?
+
+                {{ if .Values.healthchecks.uuid }}
+                  backup_status=0
+
+                  if [ $sql_snapshot_status -ne 0 ]; then
+                    echo "Reporting failure: SQL snapshot"
+                    backup_status=1
+                  elif [ $prune_status -ne 0 ]; then
+                    echo "Reporting failure: Prune"
```
Comment (lottspot) [inline kubernetes/apps/charts/postgresql-backup/templates/cronjob.yaml:79]: I personally read this error message as communicating a failure to report the status to the health checker. I would recommend an error message which reads something more like "Backup error: [restic snapshot|restic forget|etc.]"

**Review by lottspot — APPROVED** [review]
This is a nice improvement! I have one non-blocking recommendation.

File: kubernetes/apps/charts/postgresql-backup/values.yaml (hunk 1 above)
Comment (atvaccaro) [reply kubernetes/apps/charts/postgresql-backup/values.yaml:12]: I think not in this case; we're adding healthchecks as an option for deployed apps, but we'd deploy a new Cal-ITP specific instance and point the apps at that calitp.org URL in the event of a handoff.

File: kubernetes/apps/charts/postgresql-backup/templates/cronjob.yaml (hunk 2 above)
Comment (atvaccaro) [reply kubernetes/apps/charts/postgresql-backup/templates/cronjob.yaml:79]: Can do, I just took this from some other random live cluster app

**Pull request merged**